### PR TITLE
Develop a project details pop window

### PR DESCRIPTION
**Pull request by JonahKayizzi**
This pull request comprises of the following changes.

- Added a pop window for the desktop and mobile version of the portfolio
-Added the following interactions
_When the user clicks (or taps) the button to check project details, the popup with details about the project appears._
_When the user clicks (or taps) the close (X) button, the popup disappears._
-Stored projects data in a JavaScript object
-Dynamically created the projects section is created dynamically using the JavaScript object.

### CODE REVIEW

File: assets/css/styles.css [hunk 1]
```diff
@@ -557,3 +558,126 @@ button {
...
+  align-items: center;
+}
+
+#pop-section .close-wrapper {
+  width: 94%;
+  display: flex;
+  flex-direction: row;
+  justify-content: end;
+  margin-top: 1em;
+}
```
Comment (mukhammadrizooff) [inline assets/css/styles.css:580]: - [ ] [REQUIRED] Please fix layout and margin, padding errors on your desktop version modal like a [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)!

image

File: assets/css/styles.css [hunk 2]
```diff
@@ -557,3 +558,126 @@ button {
...
+.works-card-inner-pop {
+  display: flex;
+  flex-direction: row;
+  border-radius: 8px;
+  padding: 16px 0;
+  width: 100%;
+  align-items: center;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
```
Comment (mukhammadrizooff) [inline assets/css/styles.css:625]: - [ ] [REQUIRED] Please fix the mistakes in the mobile version design, I have shown the difference between the [Figma Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) and your design in the picture below!

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @JonahKayizzi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Required Changes ♻️
- [ ] [REQUIRED] Please fix the layout design on your mobile version design!

image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by JonahKayizzi** [comment]
Thanks @mukhammadrizooff  for the review.

However, I think there is a browser problem. On my browser, the app appears different from your screen shots. Kindly advise if there is a way I can show you on mine or try to view it from a different browser.

File: assets/css/styles.css (hunk 1 above)
Comment (JonahKayizzi) [reply assets/css/styles.css:580]: Hello @mukhammadrizooff , Thanks for the review. However, I see that from the figma design, the close button is just above the image. Please advise

**Review by mukhammadrizooff — COMMENTED** [review]
Hi @JonahKayizzi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 
- Perfect `UI`/`UX` design ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**